oidc_client_example/README.md

```diff
@@ -0,0 +1,7 @@
+## Start example client
+1. Build docker image via `docker build -t example_client .`
+2. Run image via `docker run -d -e CLIENT_ID=<client id> -e REDIRECT_URL=<r url> -e AUTHORITY_URL=<auth. url> -p 3000:3000 oidc_example`
```

Comment:
This is just an example and not part of our productive infrastructure, therefore it would be no correct to add it to our compose config!